src/components/BookList.js

```diff
@@ -17,22 +20,41 @@ const BookList = ({ books }) => {
   };
 
   return (
-    <ul>
+    <div className="book-list">
       {books.map((book) => (
-        <li key={book.itemId || uuidv4()}>
-          <h3>{book.title}</h3>
-          <p>
-            Author:
-            {book.author}
-          </p>
-          <p>
-            Category:
-            {book.category}
-          </p>
-          <button type="submit" onClick={() => handleRemoveBook(book.itemId)}>Remove Book</button>
-        </li>
+        <div key={book.itemId || uuidv4()} className="book-details">
+          <div className="book-info">
+            <p className="School-of Text-Style-9">
+              {book.category}
+            </p>
+            <h3 className="Title Text-Style-5">{book.title}</h3>
+            <div className="author-buttons">
+              <p className="Suzanne-Collins Text-Style-8">
+                {book.author}
+              </p>
+              <ButtonGroup variant="text" aria-label="text button group">
+                <Button type="submit">Comments</Button>
+                <Button type="submit" onClick={() => handleRemoveBook(book.itemId)}>Remove</Button>
+                <Button type="submit">Edit</Button>
+              </ButtonGroup>
+            </div>
+          </div>
+          <div className="progress">
+            <CircularProgress variant="determinate" value={75} />
+            <p>
+              75%
+              <br />
+              Completed
+            </p>
+          </div>
+          <div>
+            <p className="Current-Chapter Text-Style-7">CURRENT CHAPTER</p>
+            <p className="Current-Lesson Text-Style-4">Chapter 17</p>
+            <Button className="Rectangle-2" variant="contained" type="submit">UPDATE PROGRESS</Button>
+          </div>
```

Comment:
- [ ] Kindly fix the alignment here since I could see it is not aligned well so please make them all align to be flex-start to have one alignment like the given design 🚀💯✨ 

- [ ] Kindly change the color of update progress to be gray instead of white since it is white in your design 👍🏻

- Your
![image](https://github.com/Eccb7/book-store/assets/58553711/dfb41ef3-0027-4bc2-9764-83455b86aeb1)


- Given
![image](https://github.com/Eccb7/book-store/assets/58553711/b2a0c17a-aee4-4eb6-8a83-df088e4ef254)